t/24-worker-overall.t

```diff
@@ -75,6 +76,15 @@ $ENV{OPENQA_LOGFILE} = undef;
     }
     sub start { }
 }
+{
+    package Test::FakeCacheServiceClientInfo;
+    use Mojo::Base -base;
+    has availability_error => 'Cache service info error: Connection refused';
+}
+
+# mock cache service client
```

Comment:
this comment feels redundant to me. After all, the next line says "mock" quite explicitly already :)